### ISSUE TITLE
body_pose misalignment

### ISSUE THREAD

**Issue by sylcito** [jimmyYliu/Animatable-3D-Gaussian#12]
Hi, Great work! But I have a trouble when try animate.yaml:
python test.py --config-name animate.yaml output_path=peoplesnapshot-female-3-casual dataset=gala/turn
Report an error:
animatableGaussian/deformer/lbs.py
return torch.cat([F.pad(R, [0,0,0,1],)])
I check the error because the body_pose shape=[1, 26, 3], but it should be [1, 23, 3].
Why exist extra 3dims? Which dim should I to mask?
Thanks!

**Comment by jimmyYliu** [comment]
The two datasets use different body models. We use SMPL (with 26 bone transforms) for peoplesnapshot and gala_model for gala (with 23 bone transforms).
